__tests__/Validator.test.js

```diff
@@ -0,0 +1,48 @@
+import Validator from '../src/Validator';
+import { GAME } from '../src/Constant';
+
+describe('자동차 이름에 대한 예외처리', () => {
+  test.each([[['pobi', 'javaji']], [['a', '', 'c']], [['', '']]])(
+    `자동차 이름은 ${GAME.MIN_CAR_NAME_LENGTH}자 이상 ${GAME.MAX_CAR_NAME_LENGTH}자 이하여야한다. `,
+    async (inputs) => {
+      expect(() => {
+        Validator.validateCarNames(inputs);
+      }).toThrow('[ERROR]');
```

Comment:
toThrow('[ERROR]') 부분에 대해서 [ERROR]로 시작하는 메세지를 확인하는 방법도 좋지만, 상수로 만들어둔 메세지를 넣어주는 것도 좋을 것 같아요!

```suggestion
const expectedError = ERROR.CAR_NAME_INVALID_LENGTH;
        Validator.validateCarNames(inputs);
      }).toThrow(expectedError);
```